locale/en_US/manager.xml

```diff
@@ -547,6 +547,7 @@
 	<message key="manager.navigationMenus.unassignedMenuItems">Unassigned Menu Items</message>
 	<message key="manager.navigationMenus.form.navigationMenuArea">Active Theme Navigation Areas</message>
 	<message key="manager.navigationMenus.form.navigationMenuAreaMessage">Select a navigation area</message>
+	<message key="manager.navigationMenus.form.lastOfArea">A navigation menu is already assigned to this area.</message>
```

Comment:
Let's change the `lastOfArea` key to something that better indicates the error message. Maybe `menuAssigned`?

Reply:
Changed